README.md

```diff
@@ -1,2 +1,25 @@
 # micro_Embending-Images
 EMBEDDING IMAGES AND VIDEO
+
+Microverse project
+
+This project consists of building a Youtube clone.
+
+You can find the original project specification at:
+<https://www.theodinproject.com/courses/html5-and-css3/lessons/embedding-images-and-video>
+
+Author:
+This project was executed by Kyankya Raymond a student of Microverse.
+
+Technologies Used.
+HTML and CSS
+
+Setup
+How to setup a repo of this page on your local machine:
+
+Open your git bash and cd to the location you'd like to put your files the run the command below.
+
+Git clone: git clone <https://github.com/rkyankya/micro_Embending-Images.git>
+
+Here is the link to the live version of this project here
```

Comment:
#### `REQUIRED`

Consider adding a more descriptive readme.

Make it even more professional by adding the live link (you can use [GitHack](https://raw.githack.com/)),  screenshots of the page and what was learned during the project.